### PR TITLE
#829 - move signup button and change continue behaviour

### PR DESCRIPTION
**Pull request by rupeshparab**
Ref - https://github.com/benetech/MathShare/issues/829

Sign Up button has been moved in between sections from the top nav bar and the action on clicking continue on SignUp / SignIn page has been changed to send the user to /app in case he was to be sent to Landing Page (Default behaviour is to send the user back to the previous page)

### CODE REVIEW

**Comment by rupeshparab** [comment]
@jscholes 
Please review this PR when you get some time.

**Comment by rupeshparab** [comment]
I have added changes of https://github.com/benetech/MathShare/issues/1067 in this PR as both are related to Sign In page

**Comment by jscholes** [comment]
CC @benetechMatt and @alexrcabral

**Comment by johnhbenetech** [comment]
@alexrcabral can we get input on this?

**Comment by johnhbenetech** [comment]
@alexrcabral can you summarize what was discussed on call today and supply the revised design

**Comment by alexrcabral** [comment]
This issue has gotten overcomplicated and the original issue has conflicting comments. 

@jscholes @johnhbenetech @rupeshparab 
From clarification on the call: this is just a copy change to change the word "Sign in" to "Log in". This should occur in any existing instance of the words "Sign in". Nothing should be moved or added in this fix. 

Primary occurrences:
- the top right of the homepage (mathshare.benetech.org/#/)
- the current sign-in modal 
- the "Sign in to track your progress" message under recent sets for unauthenticated users 
- the top right of the header for the app

Screenshots & mockups to follow

**Comment by alexrcabral** [comment]
The mockups are shown in order

**Home Page**
Mockup of the top right of the home page showing the words Log in and Sign up in large dark gray font
Note that each second word ("in" and "up") should be lower case to help with visual differentiation. 

**Modal**
The login modal showing the Mathshare logo at the top, followed by the title 'Log in using' and two buttons linking to Google Classroom and Microsoft Teams. The bottom link text reads 'Continue without logging in'
Note: the primary heading is replaced with the full phrase "Log in using" with no secondary heading to address #1067. 

**Track Progress** 
Plain text reading Log in to track your problem sets, with Log in shown as an underlined link

**App Header**
Two buttons showing at the top right of the main app header, including the question mark button and the button now labelled Log in
